### ISSUE TITLE
Thing from Below cost reduction not displayed

### ISSUE THREAD

**Issue by Epix37** [HearthSim/joust#106]
Turn 6 opponent: http://hsreplay.net/games/replay/j2jCakpFoJYLcEcQNfMfF8

**Comment by beheh** [comment]
Fixed in efef906f1bb2c89f6fb496327440714fdcc81ec1.

**Comment by beheh** [comment]
Eh, I missread. Yeah, I can't do that, because I don't have the actual cost (similar to how Thaurissan ticks are not shown). At some point down the line we might implement custom rules for this for all cards like that (Frost Giant), but right now it's just too much card-specific code.

**Comment by Epix37** [comment]
Isn't this still a valid issue then? The description should just be extended to something more general.

**Comment by beheh** [comment]
Yes, but it's something I'm not willing to deal with in the near future.

**Comment by Epix37** [comment]
I disagree on `wontfix` here, this is something we should definitely fix eventually.

**Comment by beheh** [comment]
Adding to future milestone.
